AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Tests/FolderOfYAMLFilesTests.cs

```diff
@@ -19,7 +19,7 @@ public async Task ProcessFolderOfYAMLTest()
         {
             //Arrange
             //Files downloaded from repo at: https://github.com/microsoft/azure-pipelines-yaml
-            string sourceFolder = System.IO.Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location) + "/yamlFiles";
+            var sourceFolder = Path.Combine(Directory.GetCurrentDirectory(), "yamlFiles");
```

Comment:
I believe this changes the functionality somewhat, but could be wrong. I think that GetCurrentDirectory will return:

AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Tests/yamlfiles

when we need it to return:

AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Tests/**bin/**yamlfiles

Reference: https://stackoverflow.com/questions/6041332/best-way-to-get-application-folder-path

Reply:
GetCurrentDirectory is relative to the path of the exe/dll being executed, and as you have CopyToOutputDirectory in the csproj for the test yaml files - this works.